utils/lists.ts

```diff
@@ -352,6 +352,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Every adventurer needs a Dicc!",
     url: "https://etherscan.io/address/0xfecc1e1449496c0cddfb1a075e0ef74c50538c1a",
   },
+  {
+    name: "Loot Dungeon",
+    description: "D&D-inspired game that lets you fight monsters using your Loot to earn NFT rewards",
```

Comment:
Awesome. Can you please make the description shorter so it fits in one line like others? I think that should be it

Reply:
@ravindranrahul Fixed! Thanks!